docs/running/jobreport.md

```diff
@@ -0,0 +1,272 @@
+[](){#ref-jobreport}
+# Job report
+
+A batch job summary report is often requested in project proposals at CSCS to demonstrate the effective use of GPUs.
+[jobreport](https://github.com/eth-cscs/alps-jobreport/releases) is used in two stages.
+The first stage monitors an application and records the GPU usage statistics.
+The monitoring stage must be executed within a `slurm` environment.
+The information is recorded as `.csv` data within a directory  `jobreport_${SLURM_JOB_ID}` or a directory supplied on the command line.
+The second stage prints this information in a tabular form that can be inserted into a project proposal.
+
+## Downloading the job summary report
+
+Precompiled binary for the `jobreport` utility can be obtained directly from the [repository](https://github.com/eth-cscs/alps-jobreport/releases) or via the command line:
+
+```console
+$ wget https://github.com/eth-cscs/alps-jobreport/releases/download/v0.1/jobreport
+$ chmod +x ./jobreport
+```
```

Comment:
Not for this PR: @simonpintarelli since you've been deploying uenv binaries on the systems for a while, how easy do you think it would be to do the same for `jobreport`. It's a bit crazy that we need to ask users to download this tool manually.

Reply:
It should be pretty easy.
It would have to use the `/user-tools` mount so that it can be used alongside application/prgenv uenv.